### DISCUSSION TITLE
Is there a way to configure the plugins differently for different sources in a single configuration file ?

### DISCUSSION THREAD

**Discussion by gzf445** [aahnik/tgcf#161] [Q&A] [answered]
is there a way to set different plugins on each source and target?

**Comment by aahnik** [comment, marked as answer]
No, there is no way to do that in a single configuration file. For that use different configuration files.

I have opened an issue #176  for this. This feature will be added someday.

**Reply by gzf445** [reply]
in the docker version,can i run a docker with more than 1 different configuration files?

**Reply by aahnik** [reply]
you need to run different instances. you can run several different docker containers or multiple instances in a single container.
